hack/local-up-cluster.sh

```diff
@@ -714,6 +717,15 @@ function start_kubedns {
     fi
 }
 
+function start_kubedashboard {
+    if [[ "${ENABLE_CLUSTER_DASHBOARD}" = true ]]; then
+        echo "Creating kubernetes-dashboard"       
+        # use kubectl to create the dashboard
+        ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" create -f https://rawgit.com/kubernetes/dashboard/master/src/deploy/kubernetes-dashboard.yaml
```

Comment:
for reference, this is the equivalent step recommended by kubernetes/dashboard:

see: https://github.com/kubernetes/dashboard#deployment

we could keep a copy of the file in `/cluster/addons/cluster-dashboard` if you prefer instead?